src/Database/Schema/SqliteSchemaDialect.php

```diff
@@ -152,9 +152,43 @@ protected function _convertColumn(string $column): array
     }
 
     /**
-     * @inheritDoc
+     * Generate the SQL to list the tables and views.
+     *
+     * @deprecated 4.3.3 Use {@link listTablesAndViewsSql()} instead.
+     * @param array<string, mixed> $config The connection configuration to use for
+     *    getting tables from.
+     * @return array An array of (sql, params) to execute.
      */
     public function listTablesSql(array $config): array
+    {
+        return $this->listTablesWithoutViewsSql($config);
```

Comment:
> My understanding is that we want to add a new function that has consistent implementation in all drivers and a more clear name so that there's no surprises, but maintain the existing function/inconsistent implementation for backwards compatibility. 

I know we had discussed earlier being more strictly backwards compatible behavior. I don't think we need to be as concerned about that when the impacted driver is Sqlite. The number of impacted applications will be much lower.

Reply:
I can change things to remote the listTablesAndViews related code _if_ that's what you really want. If the only advantage is not being left with a redundant/deprecated function, I'd say the work is already done and it would be better to keep it as it is now. LMK if you want me to change it though, and I'll work on that.

Reply:
I can take care of those changes. Thank you for putting up with all the back and forth on this and tolerating the chaos around how many and the names of the methods we add :bow: